### PR TITLE
fix(datepicker): only show non-whitespace error text

### LINKED ISSUE #5486: Error styles shown without error text in Datepicker
When passing 2 validators that return empty strings (`validators={[afterValidator, yearValidator]}`), error styles are still shown:
image

### PR DESCRIPTION
**Pull request by redallen**
**What**: Closes #5486

**Additional issues**:

### CODE REVIEW

**Review by boaz0 — APPROVED** [review]
LGTM :+1:

File: packages/react-core/src/components/DatePicker/DatePicker.tsx [hunk 1]
```diff
@@ -199,7 +199,7 @@ export const DatePicker: React.FunctionComponent<DatePickerProps> = ({
         </div>
       </Popover>
       {helperText && <div className={styles.datePickerHelperText}>{helperText}</div>}
-      {errorText && <div className={css(styles.datePickerHelperText, styles.modifiers.error)}>{errorText}</div>}
+      {errorText.trim() && <div className={css(styles.datePickerHelperText, styles.modifiers.error)}>{errorText}</div>}
```
Comment (jschuler) [inline packages/react-core/src/components/DatePicker/DatePicker.tsx:202]: errorText can not be null or undefined?

Comment (jschuler) [reply packages/react-core/src/components/DatePicker/DatePicker.tsx:202]: OK i looked at where it's coming from, state defaults it to empty string so this is fine

**Review by jschuler — APPROVED** [review]

**Review by jessiehuff — APPROVED** [review]

**Review by dlabrecq — COMMENTED** [review]
LGTM

**Pull request merged**